src/backend/alembic/versions/3a2c90817791_.py

```diff
@@ -0,0 +1,155 @@
+"""empty message
+
+Revision ID: 3a2c90817791
+Revises: f7ad93967810
+Create Date: 2024-07-09 15:04:42.242390
+
+"""
+
+from typing import Sequence, Union
+
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision: str = "3a2c90817791"
+down_revision: Union[str, None] = "f7ad93967810"
+branch_labels: Union[str, Sequence[str], None] = None
+depends_on: Union[str, Sequence[str], None] = None
+
+
+def upgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.drop_constraint("documents_user_id_fkey", "documents", type_="foreignkey")
+    op.drop_constraint("files_conversation_id_fkey", "files", type_="foreignkey")
+    op.drop_constraint(
+        "documents_conversation_id_fkey", "documents", type_="foreignkey"
+    )
+    op.drop_constraint("files_user_id_fkey", "files", type_="foreignkey")
+    op.drop_constraint("messages_conversation_id_fkey", "messages", type_="foreignkey")
+    op.drop_constraint("messages_user_id_fkey", "messages", type_="foreignkey")
+    op.drop_constraint("snapshots_user_id_fkey", "snapshots", type_="foreignkey")
+    op.drop_constraint(
+        "snapshots_conversation_id_fkey", "snapshots", type_="foreignkey"
+    )
+    op.drop_constraint("conversations_pkey", "conversations", type_="primary")
```

Comment:
Had to manually update the migration because Alembic doesn't support updating primary keys [[ref](https://stackoverflow.com/questions/13756567/adding-primary-key-to-existing-mysql-table-in-alembic)]